### ISSUE TITLE
Can not tab into upload dialog and so cannot be read by screen reader

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#368]
![screen shot 2016-07-06 at 11 38 15](https://cloud.githubusercontent.com/assets/13200338/16615378/28814824-436e-11e6-8b98-748cf0f024f1.png)
